ui/src/dev/server/Dockerfile

```diff
@@ -2,3 +2,5 @@ FROM node:alpine
 
 # Java for Swagger client generation
 RUN apk --no-cache add openjdk8-jre
+
+RUN mkdir /.config &&  mkdir /.npm && chmod a+rwx /.config /.npm
```

Comment:
I expected to see a corresponding "Hi npm! This is where you find your stuff," line. How does it know to use this directory?